packages/tinymce5/editor_plugin.src.js

```diff
@@ -442,7 +442,7 @@ export const currentInstance = null;
       // Check if MathType editor is enabled
       if (Configuration.get('editorEnabled')) {
         // MathType button.
-        // Cmd Parameter is needed in TinyMCE4 and onAction parameter is needed in TinyMCE5.
+        // Cmd Parameter is needed in TinyMCE5 and onAction parameter is needed in TinyMCE5.
```

Comment:
This change doesn't really make sense. Maybe leave this line as-is even though we don't support TinyMCE 4 anymore.